guides/overview.md

```diff
@@ -1,33 +0,0 @@
-# A Mnesia backend for the Hammer rate-limiter
-
-[Hammer](https://github.com/ExHammer/hammer) is a rate-limiter for
-the [Elixir](https://elixir-lang.org/) language. It's killer feature is a
-pluggable backend system, allowing you to use whichever storage suits your
-needs.
-
-This package provides a Mnesia backend for Hammer, storing rate-limit counters in an Mnesia table.
-
-To get started, read
-the [Hammer Tutorial](https://hexdocs.pm/hammer/tutorial.html) first, then add
-the `hammer_backend_mnesia` dependency:
-
-```elixir
-    def deps do
-      [{:hammer_backend_mnesia, "~> 0.6"},
-      {:hammer, "~> 6.1"}]
-    end
-```
-
-... then configure the `:hammer` application to use the Redis backend:
-
-```elixir
-config :hammer,
-  backend: {Hammer.Backend.Mnesia, [expiry_ms: 60_000 * 60 * 2,
-                                    cleanup_interval_ms: 60_000 * 15]}
-```
-
-And create the Mnesia table for Hammer to use:
-
-```elixir
-Hammer.Backend.Mnesia.create_mnesia_table()
```

Comment:
where is this moved ? I think that s an important step for this backend

Reply:
GenServer init.

Reply:
https://github.com/ExHammer/hammer-backend-mnesia/pull/36#discussion_r1844079750